src/bin/lttng-sessiond/save.c

```diff
@@ -376,6 +379,119 @@ const char *get_loglevel_type_string(
 	return loglevel_type_string;
 }
 
+static
+int save_kernel_function_event(struct config_writer *writer,
+							   struct ltt_kernel_event *event)
+{
+	int ret;
+	ret = config_writer_open_element(writer, config_element_function_attributes);
```

Comment:
Add an empty line between declaration and code.

Reply:
done.